src/components/UserDetails/UserDetails.tsx

```diff
@@ -0,0 +1,14 @@
+import { UserDetailsProps } from '../../interfaces/interfaces.ts'
```

Comment:
Move `UserDetailsProps` type in this function. As discussed, the code should as close as it can to the place it is being used, that said no need to have this specific prop in another file given it will ever just be used in here.